bindings.go

```diff
@@ -0,0 +1,17 @@
+package bindings
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/pkg/errors"
+)
+
+// Config is the alias for aws config.
+type Config aws.Config
+
+// Option is used to set session's options.
+type Option func(cfg *Config)
```

Comment:
Minor: is it possible to use the underlying `aws.Config` type directly instead of creating this new type? I think the ergonomics internally (e.g. doing `(*aws.Config)(cfg)` and externally (no need to coerce into this new type) would be improved.

Reply:
What I mean by external ergonomics is that I need to convert to your `Config` type for zero reasons right? An API probably shouldn't require an action that does nothing - an action for the sake of it.

Reply:
Yeah, I agree. I had an initial idea but I guess the API would make more sense by simply require an `*aws.Config` parameter rather than `region` and `options`. I guess I wanted to hide such things to consumers but at the end of the day, since the user already has `aws-sdk-go` dep, it makes sense to let him/her use such configuration.